addons/account/models/account_move.py

```diff
@@ -3324,28 +3325,6 @@ def _compute_has_reconciled_entries(self):
         for move in self:
             move.has_reconciled_entries = len(move.line_ids._reconciled_lines()) > 1
 
-    def action_view_reverse_entry(self):
```

Comment:
don't remove method in stable version. Instead, add a comment "DEPRECATED: REMOVED IN MASTER"

Reply:
Oh yes, I forgot, it's done now.